os_builders/roles/vm_baseline/tasks/cron.yml

```diff
@@ -0,0 +1,15 @@
+
+- name: Install cron on Ubuntu
+  block:
+    - name: Install cron for Pakiti
+      apt:
+        name: "cron"
+        state: present
+        update_cache: yes
```

Comment:
Does this need to be a block if it is one task?

Reply:
It doesnt need to be, removed now